toqito/matrix_props/is_etf.py

```diff
@@ -0,0 +1,62 @@
+'''Checks if matrix forms an equilangular tight frame (ETF).'''
+import numpy as np
+from toqito.matrix_ops import vectors_to_gram_matrix
+
+def is_etf(mat: np.ndarray, rtol: float = 1e-05, atol: float = 1e-08) -> bool:
+    r"""Check if matrix constitutes an equilangular tight frame.
+    
+    Args:
+        mat(array): A matrix of any size
```

Comment:
You are missing arguments here in the description. Also, you don't need to put the type in the docstring if you supply it in the argument. Additonally, the current type is incorrect. 

You also need a period at the end of sentences.